obonet/read.py

```diff
@@ -26,6 +26,9 @@ def read_obo(path_or_file, ignore_obsolete=True):
     obo_file = open_read_file(path_or_file)
     typedefs, terms, instances, header = get_sections(obo_file)
     obo_file.close()
+    
+    if 'name' in header:
+        header['long_name'] = header.pop('name')
```

Comment:
```suggestion
        # OMRSE includes both the ontology and name key in its header
        # https://github.com/dhimmel/obonet/issues/20
        header['long_name'] = header.pop('name')
```

Reply:
Thanks for the suggestion but the way I updated it now is a bit more general and not exactly related to this issue